3회차/서주현/0_민석이의과제체크하기.py

```diff
@@ -1,3 +1,19 @@
 import sys
 
 sys.stdin = open("_민석이의과제체크하기.txt")
+
+T = int(input())
+for t in range(1, T+1) :
+    n, k = list(map(int, input().split()))
+    studentlist = list(map(int, input().split()))
+    nolist = []
+    for i in range(1, n+1) :
+        if i in studentlist :
+            continue
+        else :
+            nolist.append(i)
```

Comment:
좋은 풀이네요 👍 
`continue`를 사용하지 않는 방법도 알려드릴게요!
```suggestion
        if i not in studentlist :
            nolist.append(i)
```